source/draw/gpu/opengl/backend/gles3/Gles3ShaderProgram.ooc

```diff
@@ -86,6 +87,31 @@ Gles3ShaderProgram: class extends GLShaderProgram {
 		glUniform4f(glGetUniformLocation(this _backend, name), value x, value y, value z, value w)
 		version(debugGL) { validateEnd("ShaderProgram setUniform~FloatPoint4D") }
 	}
+	setUniform: override func ~ColorBgr (name: String, value: ColorBgr) {
+		version(debugGL) { validateStart("ShaderProgram setUniform~ColorBgr") }
+		glUniform3f(glGetUniformLocation(this _backend, name), (value blue as Float) / 255.0f, (value green as Float) / 255.0f, (value red as Float) / 255.0f)
```

Comment:
It would be nice to avoid having every possible type in setUniform but Simon did not like when I casted a color to a point and we have no generic 4D vector for GPU upload.